### PR TITLE
adding C# switch article

### PR DESCRIPTION
**Pull request by kylestarr**
### Description

I'm adding the topic of C# Switch Statements, file `/content/c-sharp/concepts/switch/switch.md`. I am currently enrolled in the iOS Developer Career Path. Contributing to the Codecademy Docs Repository is part of the curriculum.

Closes: #1733 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/switch/switch.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    break;
+  case "Michelangelo":
+    Console.WriteLine("Party dude!")
+    break;
+  default:
+    Console.WriteLine($"{favoriteTurtle} sounds cool, but I'll bet they're not a ninja!")
+    break;
+}
+
+// Output: "Does machines!"
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/switch/switch.md:70]:
```suggestion
using System;

public class Example
{
  public static void Main(string[] args)
  {
  string favoriteTurtle = "Donatello";

  switch (favoriteTurtle) 
  {
    case "Leonardo":
      Console.WriteLine("Leads!");
      break;
    case "Donatello":
      Console.WriteLine("Does machines!");
      break;
    case "Raphael":
      Console.WriteLine("Cool, but rude!");
      break;
    case "Michelangelo":
      Console.WriteLine("Party dude!");
      break;
    default:
      Console.WriteLine($"{favoriteTurtle} sounds cool, but I'll bet they're not a ninja!");
      break;
  }
 }
}

// Output: "Does machines!"
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@kylestarr I've completed my initial review. Please update the entry and tag me when you've finished making changes. Thanks

File: content/c-sharp/concepts/switch/switch.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+A `switch` statement includes one or more `case` conditions. The `case` conditions are evaluated in top-to-bottom order. If none of the `case` conditions evaluate to `true`, the `default` condition is executed. If no `default` is defined, the switch statement is bypassed.
+
+## Properties
+
+- `switch`: Definition of the `switch` statement
+- `expression`: The logic each `case` will be evaluated against
+- `case`: A value the `expression` is matched against. If the `case` matches the result of the `expression`, the code block within the `case` will be executed.
+- `break` (recommended): Terminates the switch statement. Recommended at the end of each `case` and `default`. I.e. Once the code block within a `case` has been executed, terminate the `switch` statement.
```
[the remark this answers is not in the document]
Comment (kylestarr) [reply content/c-sharp/concepts/switch/switch.md:43]: @caupolicandiaz As I understand, leaving out a `break` statement in C# will result in a compile time error. Is this OK?

**Comment by SSwiniarski** [comment]
@kylestarr please note that the email used for this commit is not attached to your GitHub account. (See the CLA notice above. Adding the e-mail to your GitHub account should resolve the issue.

File: content/c-sharp/concepts/switch/switch.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (caupolicandiaz) [reply content/c-sharp/concepts/switch/switch.md:43]: I think it's fair to keep the original "recommended" assertion. But, I think the original text was not expansive enough. It did not suggest that there are multiple ways of moving through cases. Feel free to further revise the text if you feel there is additional information, or better phrasing, to aid in explaining the functionality and outcomes of these statements.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@kylestarr Tag me when you finish making changes. Thanks

**Comment by kylestarr** [comment]
> @kylestarr please note that the email used for this commit is not attached to your GitHub account. (See the CLA notice above. Adding the e-mail to your GitHub account should resolve the issue.

@SSwiniarski Unfortunately, the original commit was done with an email I cannot link to my personal GitHub account, nor can/should I change the email address. I believe I will need to create a new PR from a different machine. I can close this PR and open a new one with @caupolicandiaz's changes incorporated.

**Pull request closed without merging**